dhis-2/dhis-api/src/main/java/org/hisp/dhis/user/User.java

```diff
@@ -233,6 +231,10 @@
 
     private FileResource avatar;
 
+    // Backward comp. field, will be removed when front-end has converted to new
+    // User model
+    private transient UserCredWrapperDto userCredentialsRaw;
```

Comment:
Maybe bit off topic but I saw this now: Could we use a less generic term name. Feel like either `Dto` or `Wrapper` would do. Maybe also go back to spelling out `Credentials` fully. I'd find `UserCredentialsDto` or UserCredentialsWrapper` would communicate better.